src/bosh-director/lib/bosh/director/core/templates/job_template_loader.rb

```diff
@@ -39,6 +39,12 @@ def process(instance_job)
         monit_erb_file = File.read(File.join(template_dir, 'monit'))
         monit_source_erb = SourceErb.new('monit', 'monit', monit_erb_file, instance_job.name)
 
+        if File.exist?(File.join(template_dir, 'properties_schema.json'))
+          properties_schema = JSON.load_file(File.join(template_dir, 'properties_schema.json'))
+        else
```

Comment:
It would be great to start a trend of moving magic strings like `'properties_schema.json'` into constants.